dom.bs

```diff
@@ -1201,6 +1201,9 @@ for discussion).
  <a>relatedTarget</a> against <var>target</var> if <var>event</var>'s <a>relatedTarget</a> is
  non-null, and null otherwise.
 
+ <li><p>If <var>event</var>'s {{Event/isTrusted}} attribute is true and <var>target</var> and
+ <var>relatedTarget</var> are identical, terminate these steps.
```

Comment:
Is this actually how we would implement this in practice? It seems somewhat weird to use the `isTrusted` attribute. Also in part because I still don't quite know when it's set and when it's not. Does `element.click()` set it?